db/migrations/notebooks/migrate_11_1_0_to_11_2_1.ipynb

```diff
@@ -139,16 +141,20 @@
   },
   {
    "cell_type": "code",
+   "execution_count": null,
    "id": "9e8a3ceb",
    "metadata": {},
-   "source": "MIGRATOR_MODULE_NAME = \"migrator_from_11_1_0_to_11_2_0\"",
    "outputs": [],
-   "execution_count": null
+   "source": [
+    "MIGRATOR_MODULE_NAME = \"migrator_from_11_1_0_to_11_2_0\""
```

Comment:
This is still correct. The module name was not modified between schema releases 11.2.0 and 11.2.1.